### PR TITLE
Improve the docs for BigQuery and GCS Action plugins

### PR DESCRIPTION
**Pull request by albertshau**
Added information to the reference docs for BigQuery source and
sink and the Google Cloud Storage create and delete actions.
Also standardized the placeholds and labels in the widgets,
and updated the description annotations.

### CODE REVIEW

File: docs/BigQueryTable-batchsource.md [hunk 1]
```diff
@@ -1,9 +1,41 @@
...
 
-This plugins exports a bigquery table as source to be ingested into the processing pipeline.
-Plugin requires a service account to access the bigquery table. In order to configure
-the service account visit https://cloud.google.com. Make sure you provide right permissions
-to service account for accessing BigQuery API.
+Description
+-----------
+This source reads the entire contents of a BigQuery table.
+BigQuery is Google's serverless, highly scalable, enterprise data warehouse.
+Data is first written to a temporary location on Google Cloud Storage,
```
Comment (gsps1) [inline docs/BigQueryTable-batchsource.md]: `Data from the bigquery table is written to`

File: docs/GCSBucketCreate-action.md [hunk 2]
```diff
@@ -1,3 +1,40 @@
-## Google Cloud Storage Create
+# Google Cloud Storage Create
 
-This plugin is used for creating directories on Google Cloud Storage (GCS).
+Description
```
Comment (gsps1) [inline docs/GCSBucketCreate-action.md:3]: unrelated to this PR : ideally these plugins should be named GCS Object Create/Delete, GCS Bucket create/delete is misleading as we are not creating/deleting buckets.

File: docs/GCSBucketCreate-action.md [hunk 3]
```diff
@@ -1,3 +1,40 @@
...
+**Project ID**: The Google Cloud Project ID, which uniquely identifies a project.
+It can be found on the Dashboard in the Google Cloud Platform Console.
+
+**Service Account File Path**: Path on the local file system of the service account key used for
+authorization. Does not need to be specified when running on a Dataproc cluster.
+When running on other clusters, the file must be present on every node in the cluster.
+
+**Bucket Name**: The bucket to create objects in.
+
+**Objects to Create**: The objects to create in the bucket.
```
Comment (gsps1) [inline docs/GCSBucketCreate-action.md:38]: will be good to give an example for this as the backend expects it in comma separated format.

**Review by gsps1 — APPROVED** [review]
minor comments, lgtm otherwise

File: docs/GCSBucketCreate-action.md (hunk 2 above)
Comment (albertshau) [reply docs/GCSBucketCreate-action.md:3]: yeah... I also thought they just create/delete buckets at first. but renaming break compatibility. I think it's ok as long as the label is correct.

File: docs/GCSBucketCreate-action.md (hunk 3 above)
Comment (albertshau) [reply docs/GCSBucketCreate-action.md:38]: For our reference docs, we're writing all new ones from the perspective of a UI user, who doesn't know anything about the underlying format.

**Pull request merged**